tools/web-fuzzing-introspection/app/webapp/routes.py

```diff
@@ -66,18 +66,23 @@ def get_coverage_report_url(project_name, datestr, language):
 
 
 def extract_introspector_raw_source_code(project_name, date_str, target_file):
+    if target_file.startswith('/'):
+        target_file = target_file[1:]
 
     if is_local:
         src_location = os.path.join(local_oss_fuzz, 'build', 'out',
-                                    project_name, 'inspector',
-                                    'source-code') + target_file
+                                    project_name, 'inspector', 'source-code',
+                                    target_file)
```

Comment:
These changes look quite suspicious -- I don't see how the semantics remain. Why take `+ target_file` into the `os.path.join`?

Reply:
Ah, I think I understand it now. So the whole point is to remove any potential `/` to make the `os.path.join` work